src/System.Windows.Forms/src/System/Windows/Forms/ToolStripItemCollection.cs

```diff
@@ -292,14 +290,14 @@ private void FindInternal(string key, bool searchAllChildren, ToolStripItemColle
 
     void IList.Clear() { Clear(); }
     bool IList.IsFixedSize { get { return ((IList)InnerList).IsFixedSize; } }
-    bool IList.Contains(object value) { return InnerList.Contains(value); }
+    bool IList.Contains(object? value) { return InnerList.Contains(value); }
     void IList.RemoveAt(int index) { RemoveAt(index); }
-    void IList.Remove(object value) { Remove(value as ToolStripItem); }
-    int IList.Add(object value) { return Add(value as ToolStripItem); }
-    int IList.IndexOf(object value) { return IndexOf(value as ToolStripItem); }
-    void IList.Insert(int index, object value) { Insert(index, value as ToolStripItem); }
+    void IList.Remove(object? value) { Remove((value as ToolStripItem)!); }
```

Comment:
would change these like direct cast that will throw if null here it self.
```suggestion
    void IList.Remove(object? value) { Remove( (ToolStripItem)(value)); }
```